tst/ProducerFunctionalityTest.cpp

```diff
@@ -198,14 +200,13 @@ TEST_F(ProducerFunctionalityTest, repeated_create_client_create_stream_put_frame
     for(i = 0; i < FUNCTIONALITY_TEST_STRESS_TEST_ITERATION; i++) {
         createDefaultProducerClient(FALSE, FUNCTIONALITY_TEST_CREATE_STREAM_TIMEOUT);
 
-        EXPECT_EQ(STATUS_SUCCESS, createTestStream(0, STREAMING_TYPE_REALTIME, TEST_MAX_STREAM_LATENCY, TEST_STREAM_BUFFER_DURATION));
+        EXPECT_EQ(STATUS_SUCCESS, createTestStream(0, STREAMING_TYPE_REALTIME, TEST_MAX_STREAM_LATENCY, TEST_STREAM_BUFFER_DURATION, FALSE));
         streamHandle = mStreams[0];
         EXPECT_TRUE(streamHandle != INVALID_STREAM_HANDLE_VALUE);
 
         for(j = 0; j < totalFrames; ++j) {
             EXPECT_EQ(STATUS_SUCCESS, putKinesisVideoFrame(streamHandle, &mFrame));
             updateFrame();
-            THREAD_SLEEP(mFrame.duration);
```

Comment:
So, removing thread sleep might change the temporal semantics of the test. Could you make sure we are not changing the behavior with this? If we produce frames at the CPU speed, we might fill the buffer immediately - is this what's intended in all these changes?

Reply:
I only removed the sleep in places where we are just simulating some normal streaming (no fault injection or anything else). The only change it will have is changing the timing of acks coming in, which I think is ok.